src/util/token.js

```diff
@@ -3,17 +3,9 @@ const crypto = require('crypto') // sodium doesn't have HMAC
 const base64url = require('base64url')
 
 const TOKEN_HMAC_INPUT = 'token'
+const AUTH_HMAC_INPUT = 'authorization'
 
-const token = (seed, publicKey) => {
-  // seed and public key should be stored as base64url strings
-  const seedBuffer = base64url.toBuffer(seed)
-  const publicKeyBuffer = base64url.toBuffer(publicKey)
-
-  const sharedSecretBuffer = tweetnacl.scalarMult(
-    crypto.createHash('sha256').update(seedBuffer).digest(),
-    publicKeyBuffer
-  )
-
+const tokenFromSharedSecret = (sharedSecretBuffer) => {
```

Comment:
I think it would be better to export this function explicitly from the if it's meant to be part of the API